### ISSUE TITLE
Creation date not shown in Dolphin file manager

### ISSUE THREAD

**Issue by digitalsignalperson** [NixOS/nixpkgs#68334]
**Describe the bug**

In Dolphin, adding a column for "Created" date does not show anything in the column.

This is supposed to be fixed as of KDE Frameworks 5.58: 

> With KDE Frameworks 5.58, the Dolphin file manager and other KDE applications will finally begin displaying file creation dates/times as a long sought after feature on Linux systems. 
(https://www.phoronix.com/scan.php?page=news_item&px=KDE-Finally-File-Creation-Times)

KInfoCenter indicates KDE Frameworks Version 5.61.0. 

**To Reproduce**
1. Use nixos-unstable channel
2. Enable plasma5 desktop and sddm

**Expected behavior**
Creation date should be shown. Can confirm it works as expected on an Arch Linux install since 5.58

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.69, NixOS, 19.09pre191617.4e60699fa72 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.09pre191617.4e60699fa72"`
 - channels(username): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: [ plasma5 dolphin kdeFrameworks ]
# a list of nixos modules affected by the problem
module:
```

**Comment by matthewbauer** [comment]
This is probably due to using glibc 2.27 still. Someone should try bumping to glibc 2.28 and see if this fixes things.

**Comment by digitalsignalperson** [comment]
Ah, so this pull request might solve it: https://github.com/NixOS/nixpkgs/pull/66528

**Comment by digitalsignalperson** [comment]
When glibc gets bumped, will kdeFrameworks need to be rebuilt?

Looks like the statx detection is done at compile time https://phabricator.kde.org/D20096

**Comment by matthewbauer** [comment]
> When glibc gets bumped, will kdeFrameworks need to be rebuilt?

Yeah, everything needs to get rebuilt, which is why it takes a little while to get things in order. I forgot to mention, you can follow the progress here:

https://github.com/NixOS/nixpkgs/pull/66528

**Comment by matthewbauer** [comment]
Should be fixed with 447edaa32fcee706be24db4389f4759fad68a785 in master.

**Comment by matthewbauer** [comment]
(feel free to reopen if this is not the case).
